pystac/extensions/sar.py

```diff
@@ -318,7 +318,7 @@ def ext(cls, obj: T, add_if_missing: bool = False) -> "SarExtension[T]":
             return cast(SarExtension[T], AssetSarExtension(obj))
         else:
             raise pystac.ExtensionTypeError(
-                f"SAR extension does not apply to type {type(obj)}"
+                f"SAR extension does not apply to type '{type(obj).__name__}'"
```

Comment:
It looks like this change could also be made for a number of other extensions as well. Is there a reason to only change this for the `sar` and `view` extensions, or should we update the `ext` method for all extensions in this PR?

Reply:
I can do that, but I'd need to be sure that you're OK with changing all of them so it's not wasted effort. What do you think?